src/Sentry/Laravel/EventHandler.php

```diff
@@ -255,16 +249,56 @@ protected function logHandler($level, $message, $context)
      * @param \Illuminate\Log\Events\MessageLogged $logEntry
      */
     protected function messageLoggedHandler(MessageLogged $logEntry)
+    {
+        $this->addLogBreadcrumb($logEntry->level, $logEntry->message, $logEntry->context);
+    }
+
+    /**
+     * Helper to add an log breadcrumb.
+     *
+     * @param  string $level   Log level. May be any standard.
+     * @param  string $message Log messsage.
+     * @param  array  $context Log context.
+     * @return void
+     */
+    private function addLogBreadcrumb(string $level, string $message, array $context = []): void
     {
         Integration::addBreadcrumb(new Breadcrumb(
-            $logEntry->level,
+            $this->logLvlToBreadcrumbLvl($level),
```

Comment:
```suggestion
            $this->logLevelToBreadcrumbLevel($level),
```

No need to shorten this IMHO.